examples/api/lib/cupertino/modal_bottom_sheet/modal_bottom_sheet.0.dart

```diff
@@ -0,0 +1,346 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
```

Comment:
Should you do this example with CupertinoApp etc.? Or maybe MaterialPageRoute won't work...

Reply:
MaterialApp is a little simpler for mixing together Material and Cupertino transitions in one app. Because MaterialPageRoute looks at the Material theme, it gets a little funky trying to do that in a CupertioApp. The actual MBS widget would have an example in CupertinoApp without the Material transitions mixing.